src/index.ts

```diff
@@ -66,37 +90,129 @@ export class MtLinkSdk {
       mode: validModes.indexOf(mode) === -1 ? 'production' : mode
     };
   }
-
+  /**
+   * Use this method to send a guest identifier to Moneytree so that Moneytree can forward it as `saml_subject_id` parameter
+   * to the Identity Provider (IdP) via the SAMLRequest during SAML SSO flows. See the [SAML SSO documentation for details](https://docs.link.getmoneytree.com/docs/saml-introduction#saml-subject-identifier).
+   *
+   * This parameter can be set during {@link init} or changed on a request-by-request basis with this method.
+   * The `saml_subject_id` parameter will be forwarded to the `authorize`, `logout` and `open-service` methods when defined.
+   */
   public setSamlSubjectId(value: string): void {
     this.storedOptions.samlSubjectId = value;
   }
 
+  /**
+   * OAuth authorization method to request guest consent to access data via the [Link API](https://getmoneytree.com/jp/link/about).
+   *
+   * The only supported flow is authorization code grant with PKCE [PKCE](https://auth0.com/docs/flows/concepts/auth-code-pkce)
+   * If the user is not logged in yet this will show the login screen and redirect the guest to the consent screen after they log in.
+   * After the guest consents, they will be redirected to the redirect URI with an authorization code.
+   * If the user is already logged and has granted consent in the redirection happens immediately.
```

Comment:
Line 109: "guest"
Line 110: "user"

I don't know which word is canonical, but I think we should stick to one.

Reply:
I replaced all `guest` with `user` in the docs.
Some of our APIs still have guest (like the scope `guest_read` and some API response objects).
But this is 99% consistent now.

Reply:
and yes, logged in is better 😁